src/test/java/study/SetCollectionTest.java

```diff
@@ -0,0 +1,50 @@
+package study;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+import org.junit.jupiter.params.provider.ValueSource;
+
+import java.util.HashSet;
+import java.util.Set;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class SetCollectionTest {
+    private Set<Integer> numbers;
+
+    @BeforeEach
+    void setUp(){
+        numbers = new HashSet<>();
+        numbers.add(1);
+        numbers.add(1);
+        numbers.add(2);
+        numbers.add(3);
+    }
+
+    @Test
+    void 컬렉션크기_확인_테스트(){
+        int size = numbers.size();
+
+        assertThat(size).isEqualTo(3);
+    }
+
+    @ParameterizedTest
+    @ValueSource(ints = {1, 2, 3})
+    void 해시셋_ValueSource_테스트(int input){
+        assertThat(numbers.contains(input)).isTrue();
```

Comment:
```suggestion
        assertThat(numbers).contains(input);
```

collection의 경우 위와 같이 구현도 가능함